experiment/aim.md

```diff
@@ -1,6 +1,9 @@
 Infrared region lies between the visible and microwave regions of the electromagnetic spectrum and works mostly based on absorption spectroscopy. The infrared portion of the electromagnetic spectrum is usually divided into three regions; the near or higher energy IR, mid IR and far or low energy infrared. The mid-infrared which is (4000 - 400 cm-1 ) gives the fundamental vibrations and associated rotational-vibrational structure of the substance under study. Infrared spectroscopy is an absorption of infrared radiation by the molecules and is used to elucidate the structure of the molecule by identifying their functional groups. One of the common laboratory instruments that use this technique is a Fourier transform infrared (FTIR) spectrometer.
 
-[image](images/figure1_exp6.gif)
+<image src="images/figure1_exp6.gif">
 
 Figure 1: Energy levels of molecules.
-Picture source: [ http://www.wag.caltech.edu/home/jang/genchem/infrared.htm](http://www.wag.caltech.edu/home/jang/genchem/infrared.htm)
\ No newline at end of file
+
+
+(i) To learn sample prepartion and handling procedures to measure an IR spectrum using salt plates.
+(ii) Spectral analysis of the sample measured.
```

Comment:
In the deployed experiment on the testing branch, these 2 points are coming on the same line